scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -49,17 +51,21 @@ def __init__(self,
                 test. This is the main case associated with the test.
             separate_builds (bool): Whether separate builds are needed for the
                 two cases. If False, case2 uses the case1 executable.
+            separate_builds (bool): Whether separate runs are needed for the
+                two cases. If False, case2 uses the case1 run area. Can be useful for restart tests
             run_two_suffix (str, optional): Suffix appended to the case name for
                 the second run. Defaults to 'test'. This can be anything other
                 than 'base'.
             run_one_description (str, optional): Description printed to log file
                 when starting the first run. Defaults to ''.
             run_two_description (str, optional): Description printed to log file
                 when starting the second run. Defaults to ''.
+            multisubmit (bool): Do first and second runs as different submissions
```

Comment:
Thanks for the clarification about rerunning. I'm still confused about how these tests work with the system_tests_compare_two infrastructure. Are you able to describe how this works? Specifically: It seems like both the base case and the restart case would have RESUBMIT set to 1. After the run of the base case, RESUBMIT would be decremented in the base case and the case would be resubmitted. Then the logic would cause it to execute the restart case. But then what happens with respect to RESUBMIT? Maybe the check for RESUBMIT happens in the base case, where it has already been decremented to 0, and so the RESUBMIT=1 value is ignored in the restart case? I could believe that. But if that's what's actually happening, it should be documented somewhere, because it's confusing.